template.md

```diff
@@ -0,0 +1,305 @@
+
+To link to an external file, use the full URL as the link.
+
+- Example: [GitHub](http://www.github.com)
+
+If a URL appears in a Markdown file, it will be transformed into a clickable link.
+
+- Example: http://www.github.com
+
+## Lists
+
+### Ordered lists
+
+1. This 
+1. Is
+1. An
+1. Ordered
+1. List  
+
+
+#### Ordered list with an embedded list
+
+1. Here
+1. comes
+1. an
+1. embedded
+    1. Miss Scarlett
+    1. Professor Plum
+1. ordered
+1. list
+
+
+### Unordered Lists
+
+- This
+- is
+- a
+- bulleted
+- list
+
+
+##### Unordered list with an embedded list
+
+- This 
+- bulleted 
+- list
+    - Mrs. Peacock
+    - Mr. Green
+- contains  
+- other
+    1. Colonel Mustard
+    1. Mrs. White
+- lists
+
+
+## Horizontal rule
+
+---
+
+## Tables
+
+| Tables        | Are           | Cool  |
+| ------------- |:-------------:| -----:|
+| col 3 is      | right-aligned | $1600 |
+| col 2 is      | centered      |   $12 |
+| col 1 is default | left-aligned     |    $1 |
+
+
+## Code
+
+### Generic codeblock
+
+Indent code four spaces for generic codeblock coding.
+
+    function fancyAlert(arg) {
+      if(arg) {
+        $.docs({div:'#foo'})
+      }
+    }
+
+
+### Codeblocks with language identifier
+
+Use three backticks (&#96;&#96;&#96;) + a language ID to apply language-specific color coding to a code block.  Here is the entire list of [GitHub Flavored Markdown (GFM) language IDs](https://github.com/jmm/gfm-lang-ids/wiki/GitHub-Flavored-Markdown-(GFM)-language-IDs).
+
+##### C&#9839;
+
+```c#
+using System;
+namespace HelloWorld
+{
+    class Hello 
+    {
+        static void Main() 
+        {
+            Console.WriteLine("Hello World!");
+
+            // Keep the console window open in debug mode.
+            Console.WriteLine("Press any key to exit.");
+            Console.ReadKey();
+        }
+    }
+}
+```
+#### Python
+
+```python
+friends = ['john', 'pat', 'gary', 'michael']
+for i, name in enumerate(friends):
+    print "iteration {iteration} is {name}".format(iteration=i, name=name)
+```
+#### PowerShell
+
+```powershell
+Clear-Host
+$Directory = "C:\Windows\"
+$Files = Get-Childitem $Directory -recurse -Include *.log `
+-ErrorAction SilentlyContinue
+```
+
+### Inline code
+
+Use backticks (&#96;) for `inline code`.
+
+## Blockquotes
+
+> The drought had lasted now for ten million years, and the reign of the terrible lizards had long since ended. Here on the Equator, in the continent which would one day be known as Africa, the battle for existence had reached a new climax of ferocity, and the victor was not yet in sight. In this barren and desiccated land, only the small or the swift or the fierce could flourish, or even hope to survive.
+
+## Images
+
+### Static Image
+
+![this is the alt text](./images/Logo_DotNet.png)
+
+### Linked Image
+
+[![alt text for linked image](./images/Logo_DotNet.png)](https://dot.net) 
+
+### Animated gif
+
+![animated gif](https://github.com/Microsoft/Docs/tree/master/media/hololens.gif)
```

Comment:
I'll remove this one and combine with the static image link since they use the same syntax